docs/DeveloperGuide.adoc

```diff
@@ -64,13 +64,24 @@ image::LogicClassDiagram.png[]
 [discrete]
 ==== How the architecture components interact with each other
 
-The _Sequence Diagram_ below shows how the components interact with each other for the scenario where the user issues the command `delete 1`.
+The _Sequence Diagram_ below shows how the components interact with each other for the scenario where the user issues the command `delete 1 i/ ski`.
+
+.Component interactions for `delete 1 i/ ski` command
 
-.Component interactions for `delete 1` command
 image::ArchitectureSequenceDiagram.png[]
```

Comment:
The Save to file action should be changed to its method name. Furthermore, the arrow is detached from the activation bar, should be touching instead.

Reply:
Out of curiosity, why are there UML notes for your sequence diagram? Are these notes really helpful in this case, or can the reader get the same information from your descriptive method names?